### PR TITLE
Update dex-contracts to fix side effects when loading orderbook

### PR DESCRIPTION
**Pull request by fleupold**
This PR updates the dex-contracts dependency and adds an e2e test to make sure two consecutive orderbook fetches are returning the same result. This is to verify there is no side-effects when computing the transitive orderbook.

### CODE REVIEW

**Review by josojo — APPROVED** [review]
Good new unit test! Easy check for a piece of mind

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     }
   },
   "dependencies": {
-    "@gnosis.pm/dex-contracts": "^0.2.6",
+    "@gnosis.pm/dex-contracts": "^0.2.8",
```
Comment (bh2smith) [inline package.json:30]: What happened to 7? Or is it just unlucky.

File: src/index.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ const argv = yargs
   })
   .option('page-size', {
     describe: 'The number of orders to fetch per page',
-    default: 50,
+    default: 100,
```
Comment (bh2smith) [inline src/index.ts:18]: Any particular reason for the bump here?

**Review by bh2smith — APPROVED** [review]
Looks fine, but the only real changes I see here are the introduction of a test. Does this PR not actually fix anything but only include a test proving that the problem does not occur?

File: package.json (hunk 1 above)
Comment (fleupold) [reply package.json:30]: It was a performance boost which was never included in this repo

File: src/index.ts (hunk 2 above)
Comment (fleupold) [reply src/index.ts:18]: performance boost in the way we query the orderbook

**Pull request merged**